fedimint-dbtool/src/dump.rs

```diff
@@ -19,8 +19,9 @@ use fedimint_gateway_server::Gateway;
 use fedimint_rocksdb::RocksDbReadOnly;
 use fedimint_server::config::ServerConfig;
 use fedimint_server::config::io::read_server_config;
-use fedimint_server::consensus::db as ConsensusRange;
```

Comment:
This alias made no sense, and was causing issues now that prefix moved to main `db.rs`